src/textual/widgets/_data_table.py

```diff
@@ -170,14 +171,18 @@ class Column:
     content_width: int = 0
     auto_width: bool = False
 
-    @property
-    def render_width(self) -> int:
-        """Width in cells, required to render a column."""
-        # +2 is to account for space padding either side of the cell
-        if self.auto_width:
-            return self.content_width + CELL_X_PADDING
-        else:
-            return self.width + CELL_X_PADDING
+    def render_width(self, data_table: DataTable[Any]) -> int:
```

Comment:
Since this is no longer a property, it should be named "<VERB>_<NOUN>".  Suggest "get_render_width".

Reply:
That's reasonable. See b84334c.